llvm/test/CodeGen/X86/masked_expandload_isel.ll

```diff
@@ -0,0 +1,33 @@
+; RUN: llc < %s -mtriple=x86_64-unknown-linux-gnu -start-after=codegenprepare -stop-before finalize-isel | FileCheck %s
```

Comment:
why do you need -start-after?

Reply:
Wouldn't that not happen if it's legal for the target?

Reply:
Use update_mir_test_checks.py. I'll fix the compress store test to use the script and we can rebase this on top of that

Reply:
Please rebase after bdfebc310e67c8e96909e155669277ab75d784ba